site/docs/pages/token/token-search.mdx

```diff
@@ -0,0 +1,53 @@
+import { TokenSearch } from '../../../../src/token/components/TokenSearch';
+import App from '../App';
+import { tokens } from '../../components/tokens';
+
+# `<TokenSearch />`
+
+### Warning: component under development, may change in coming weeks
+
+The `<TokenSearch />` component allows users to search on a given list of tokens by name, symbol or address with an optional debounce delay.
+
+## Usage
+
+:::code-group
+
+```tsx [code]
+import { TokenSearch } from '@coinbase/onchainkit/token';
+
+const tokens = [
+  // ...
+];
+
+<TokenSearch tokens={tokens} onSearch={(tokens) => {}} />; // [!code focus]
```

Comment:
A tricky thing here, is that we should make it easier to do both:
1. API calls of different tokens
2. Use the data that is found